api/routers/v1/timelog.py

```diff
@@ -46,18 +54,26 @@ async def add_template(template: TemplateNewSchema, db: Session = Depends(get_db
     \f
     :param item: User input.
     """
+    if template.is_global and "manager" not in user.roles:
+        raise HTTPException(status_code=403, detail="You are not authorized to create global templates")
+    else:
+        if template.user_id is not None and template.user_id is not user.id:
```

Comment:
Same here:

```suggestion
        if template.user_id is not None and template.user_id != user.id:
```

and to simplify:

```suggestion
        if template.user_id and template.user_id != user.id:
```